relayer/pkg/chainlink/txm/txm_test.go

```diff
@@ -32,34 +33,37 @@ func TestIntegration_Txm(t *testing.T) {
 	require.NoError(t, err)
 
 	// parse keys into expected format
-	localKeys := map[string]keys.Key{}
+	localKeys := map[string]*big.Int{}
 	localAccounts := map[string]string{}
 	for i := range accounts {
 		privKey, err := caigotypes.HexToBytes(accounts[i].PrivateKey)
 		require.NoError(t, err)
 
-		key := keys.Raw(privKey).Key()
-		assert.Equal(t, caigotypes.HexToHash(accounts[i].PublicKey), caigotypes.HexToHash(key.ID()))
-		localKeys[key.ID()] = key
-		localAccounts[key.ID()] = accounts[i].Address
+		senderAddress := caigotypes.HexToHash(accounts[i].PublicKey).String()
+		//assert.Equal(t, caigotypes.HexToHash(accounts[i].PublicKey), caigotypes.HexToHash(key.ID()))
```

Comment:
Drop or restore?

Reply:
are you getting pinged to review this over and over? i've been battling the code all day and it's been WIP. i thought you would only get pinged if i manually request review.

going forward would you prefer i put it back in draft when i'm pushing a lot of changes that i don't expect to be final?

Reply:
NBD I was just trying to stay ahead since we are blocked on the error signature change due to an odd testing module dependency between this repo and solana.
Draft could still be helpful though :shrug: